accumulo-bootstrap.sh

```diff
@@ -14,9 +14,11 @@ export PATH=$PATH:/usr/lib/accumulo/bin:$JAVA_HOME/bin
 
 echo "Installing Accumulo"
 
-curl -O -L -s http://mirrors.koehn.com/apache/accumulo/1.7.0/accumulo-1.7.0-bin.tar.gz
-tar xvzf accumulo-1.7.0-bin.tar.gz
-sudo mv accumulo-1.7.0 /usr/lib/accumulo
+ACC_VERSION=1.7.1
+
+curl -O -L -s https://archive.apache.org/dist/accumulo/$ACC_VERSION/accumulo-$ACC_VERSION-bin.tar.gz
```

Comment:
The issue seems to be that mirrors only keep the latest patch version(s). In order to anker the version, the url needs to point to the archives.